app/src/main/java/com/codesoom/assignment/ProductNotFoundException.java

```diff
@@ -1,5 +1,8 @@
 package com.codesoom.assignment;
 
+/**
+ * 사용자를 찾을 수 없는 경우 던지는 예외입니다
+ */
```

Comment:
이 정도가 적절하겠죠. (마침표를 잊지 마세요. javadoc은 외부에 공개하는 문구라고 생각해 보세요.)
```suggestion
/**
 * 사용자를 찾을 수 없는 경우 던집니다.
 */
```